LuckParser/Models/BossLogic/Sabetha.cs

```diff
@@ -166,8 +169,49 @@ public override void ComputeAdditionalBossData(Boss boss, ParsedLog log)
                     }
                     break;
                 case (ushort)Kernan:
+                    List<CastLog> bulletHail = cls.Where(x => x.SkillId == 31721).ToList();
+                    Boss kernan = Targets.FirstOrDefault(x => x.ID == (ushort)Kernan);
```

Comment:
You should simply be using boss or replay here no? Why look for targets again when everything's already here?